### PR TITLE
Changed the text color of the footer email and tel numbers to white

### PR DESCRIPTION
**Pull request by SilunaP**
## Purpose
The purpose of this Pull Request is to fix issue #67

## Goals
Changed color of footer email and tel numbers

## Approach
With the use of CSS

### Screenshots
image

  
##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x ] I have read and understood the development best practices guidelines
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by YoshithaRathnayake** [comment]
@SilunaP I think you have to resolve the conflicts right?

**Review by YoshithaRathnayake — COMMENTED** [review]
@SilunaP I think you have to resolve the conflicts right?

File: assets/css/custom.css [hunk 1]
```diff
@@ -229,3 +225,11 @@ color: black; }
...
     box-shadow: 0 0 20px rgba(0, 0, 0, 0.541);
 }
+
+.e-mail, .telno-1, .telno-2, .telno-3 {
+    color: white;
+}
+
+.e-mail:hover, .telno-1:hover, .telno-2:hover, .telno-3:hover {
+    color: rgb(221, 221, 221);
+}
```
Comment (YoshithaRathnayake) [inline assets/css/custom.css]: @SilunaP brother you can add a line at the bottom of this file, if not the last line that means } will loss

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -235,11 +235,11 @@ <h6 class="text-uppercase fw-bold mb-4">Contact</h6>
             <p><i class="fas fa-home me-3 text-secondary"></i> Kandy, Sri Lanka</p>
             <p>
               <i class="fas fa-envelope me-3 text-secondary"></i>
-              <a href="mailto:carexnextgen@gmail.com" class="text-info">carexnextgen@gmail.com</a>
+              <a href="mailto:carexnextgen@gmail.com" class="e-mail">carexnextgen@gmail.com</a>
```
Comment (YoshithaRathnayake) [inline index.html]: Use bootstrap for this without custom css bcs it is easy @SilunaP brother.

- Remove the e-mail and telno-1,2,3 class and put text-light

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (SilunaP) [reply index.html]: ok

Comment (YoshithaRathnayake) [reply index.html]: 👍

File: assets/css/custom.css (hunk 1 above)
Comment (YoshithaRathnayake) [reply assets/css/custom.css]: Can you do this also @SilunaP brother

**Pull request merged**